packages/client/src/builders/search/facetBuilder.ts

```diff
@@ -357,6 +358,29 @@ export class FacetBuilder {
 
         return this;
     }
+
+    public addRecentlyPurchasedFacet(
+        purchaseQualifiers: PurchaseQualifiers,
+        selectedValues: boolean[] | null = null,
+        facetSettings?: FacetSettings): this {
+
+        const facet: RecentlyPurchasedFacet = {
+            $type: 'Relewise.Client.DataTypes.Search.Facets.Queries.RecentlyPurchasedFacet, Relewise.Client',
+            field: 'Data',
+            settings: facetSettings,
+            selected: selectedValues,
+            purchaseQualifiers: {
+                byUser: purchaseQualifiers.byUser ?? false,
+                byUserCompany: purchaseQualifiers.byUserCompany ?? false,
+                byUserParentCompany: purchaseQualifiers.byUserParentCompany ?? false,
+                sinceMinutesAgo: purchaseQualifiers.sinceMinutesAgo ?? 1234,
```

Comment:
Is there a reason for these fallbacks? Values are required right? 
The `1234` seems like test code :)